python/modules/LLObjectsProducer.py

```diff
@@ -0,0 +1,208 @@
+	self.Jet_Stop0l30    = map(self.SelJets30, jets)
+	local_BJet_Stop0l    = map(self.SelBtagJets, jets)
+        self.BJet_Stop0l     = [a and b for a, b in zip(self.Jet_Stop0l, local_BJet_Stop0l )]
+	mt 		     = self.SelMtlepMET(electrons, muons, isotracks, met)
+	countIskLep 	     = sum([(i.Stop0l and (abs(i.pdgId) == 11 or abs(i.pdgId) == 13)) for i in isotracks])
+	countIskHad 	     = sum([(i.Stop0l and abs(i.pdgId) == 211) for i in isotracks])
+	countIskHad_ptgeq20  = sum([(i.Stop0l and abs(i.pdgId) == 211 and i.pt > 20) for i in isotracks])
+	countEle	     = sum([e.Stop0l for e in electrons])
+	countMuon	     = sum([m.Stop0l for m in muons])
+	sortedPhi 	     = self.GetJetSortedIdx(jets)
+        PassdPhiMedDM        = self.PassdPhi(sortedPhi, [0.15, 0.15, 0.15], invertdPhi=True)
+	dphiISRMet	     = abs(deltaPhi(fatjets[stop0l.ISRJetIdx].phi, met.phi)) if stop0l.ISRJetIdx >= 0 else -1
+	self.Tau_Stop0l      = map(self.SelTauPOG, tau)
+	countTauPOG	     = sum(self.Tau_Stop0l)
+	if not self.isData:
+		self.GenVisTau_Stop0l= map(self.SelGenTau, gentau)
+		countGenTau	     = sum(self.GenVisTau_Stop0l)
+		countGenTau_pt10to20  =sum([g.pt > 10 and g.pt < 20 and gt for g, gt in zip(gentau, self.GenVisTau_Stop0l)])
+		countGenTau_ptgeq20  = sum([g.pt > 20 and gt for g, gt in zip(gentau, self.GenVisTau_Stop0l)])
+		countGenTau_1Prong0PiZero  = sum([g.status == 0  and gt for g, gt in zip(gentau, self.GenVisTau_Stop0l)])
+		countGenTau_1Prong1PiZero  = sum([g.status == 1  and gt for g, gt in zip(gentau, self.GenVisTau_Stop0l)])
+		countGenTau_1Prong2PiZero  = sum([g.status == 2  and gt for g, gt in zip(gentau, self.GenVisTau_Stop0l)])
+		countGenTau_3Prong0PiZero  = sum([g.status == 10 and gt for g, gt in zip(gentau, self.GenVisTau_Stop0l)])
+		countGenTau_3Prong1PiZero  = sum([g.status == 11 and gt for g, gt in zip(gentau, self.GenVisTau_Stop0l)])
+		countGenTau_other	   = sum([g.status == 15 and gt for g, gt in zip(gentau, self.GenVisTau_Stop0l)])
+
+        ### Store output
+	self.out.fillBranch("Stop0l_nJets_HighPt",    	sum(self.Jet_Stop0lHighPt))
+	self.out.fillBranch("Stop0l_nJets30",    	sum(self.Jet_Stop0l30))
+	self.out.fillBranch("Stop0l_nbtags_Loose",   	sum(self.BJet_Stop0l))
+	self.out.fillBranch("Stop0l_MtLepMET",  	mt)
+	self.out.fillBranch("nLeptonVeto",    		countMuon + countEle + countIskLep)
+	self.out.fillBranch("Stop0l_nIsoTracksLep",	countIskLep)
+	self.out.fillBranch("Stop0l_nIsoTracksHad",	countIskHad)
+	self.out.fillBranch("Stop0l_nIsoTracksHad_ptgeq20",	countIskHad_ptgeq20)
+	self.out.fillBranch("Stop0l_nVetoElec", 	countEle)
+	self.out.fillBranch("Stop0l_nVetoMuon", 	countMuon)
+	self.out.fillBranch("Stop0l_nVetoElecMuon", 	countEle + countMuon)
+	self.out.fillBranch("Pass_dPhiMETMedDM", 	PassdPhiMedDM)
+	self.out.fillBranch("Stop0l_dPhiISRMET",	dphiISRMet)
+	self.out.fillBranch("Stop0l_TauPOG",		countTauPOG)
+	if not self.isData:
+		self.out.fillBranch("Stop0l_GenVisTau",		countGenTau)
```

Comment:
Since we moved on with TauMVA, do we still need these?

Reply:
No we don't need the stop0l_genVisTau variables. They just happened to be in the file. I'll clean it up and redo the PR